docs/content/UnderlineNav2.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: UnderlineNav
+status: Draft
+description: Use an underlined nav to allow tab like navigation with overflow behaviour in your UI.
+---
+
+## Examples
+
+### Simple
+
+```jsx live drafts
+<UnderlineNav label="simple nav">
+  <UnderlineNav.Link selected>Item 1</UnderlineNav.Link>
+  <UnderlineNav.Link>Item 2</UnderlineNav.Link>
+  <UnderlineNav.Link>Item 3</UnderlineNav.Link>
+</UnderlineNav>
+```
+
+### With icons
+
+```jsx live drafts
+<UnderlineNav label="simple nav with icons">
+  <UnderlineNav.Link selected leadingIcon={EyeIcon}>
+    Item 1
+  </UnderlineNav.Link>
+  <UnderlineNav.Link>Item 2</UnderlineNav.Link>
+</UnderlineNav>
+```
+
+## Props
+
+### UnderlineNav
+
+<PropsTable>
+  <PropsTableRow name="aria-label" type="string" />
+  <PropsTableRow name="aria-labelledby" type="string" />
+  <PropsTableRow name="aria-describedby" type="string" />
+  <PropsTableRow
+    name="overflow"
+    type="'auto' | 'menu' | 'scroll'"
+    description="Controls the type of overflow behaviour in smaller screens"
+  />
+  <PropsTableRow name="align" type="right | left" description="The alignment of the nav links" />
+  <PropsTableRow
+    name="afterSelect"
+    type="(event) => void"
+    description="The handler that gets called when a nav link child is selected"
+  />
```

Comment:
I think this needs to be updated with the new `onSelect` prop info

Reply:
So the `onSelect` is on `UnderlineNavLink`, and the `afterSelect` is on `UnderlineNav`. I found this pattern in `ActionList`. Do you think we need to change it?

Reply:
We can keep afterSelect - just update the `UnderlineNavProps` type. Currently, I'm seeing `onSelect` and not `afterSelect`